scipy/optimize/_linprog_highs.py

```diff
@@ -211,7 +212,7 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
         'simplex_dual_edge_weight_strategy',
         choices={'dantzig': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_DANTZIG,
                  'devex': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_DEVEX,
-                 'steepest-devex': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STEEP2DVX,
+                 'steepest-devex': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE,
```

Comment:
Did they remove steep2dvx? Is this equivalent?

Reply:
This is equivalent to what was there before, this was a renaming of the enumeration.  This is verified by checking the [source](https://github.com/ERGO-Code/HiGHS/blob/25c112b29520b4ecb1224e33c1f42471dbe51cbe/src/simplex/HDual.cpp#L1919) where `allow_dual_steepest_edge_to_devex_switch` is set to `true`